util/test/checkChplInstall

```diff
@@ -161,9 +161,6 @@ fi
 chpl_comm="$($CHPL_HOME/util/chplenv/chpl_comm.py)"
 chpl_launcher="$($CHPL_HOME/util/chplenv/chpl_launcher.py)"
 
-# Necessary to suppress warnings when WARNINGS=1 (e.g. nightly build settings)
```

Comment:
Based on the comment, I wonder if this was meant to be throwing `--no-cc-warnings`

Reply:
It could be, but given that we haven't been throwing it, and it's been passing in nightly, it doesn't seem like it must be necessary to do so to me (and it always seems better to me to inherit such choices as much as possible rather than to pin them in the script itself).

My guess is that since this was being done as part of moving from `start_test`-based `make check` testing to something standalone, Ben may have noted that start_test throws `--cc-warnings` and so added it here to preserve the same behavior.  But that really is just a guess.

Reply:
That's a good point